node.js/constants.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
This is up to you but I find pulling out constants that are only used in a single file is less clean.